specification/subscription/resource-manager/Microsoft.Subscription/preview/2020-01-01-preview/subscriptions.json

```diff
@@ -0,0 +1,469 @@
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Subscription/SupportPlan/default": {
+      "get": {
+        "operationId": "SupportPlanDefault_Get",
+        "x-ms-examples": {
+          "SupportPlan": {
+            "$ref": "./examples/SupportPlanDefault.json"
+          }
+        },
+        "description": "The operation to find out the support plan",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/DefaultSupportPlanResponseResult"
+            }
+          },
+          "default": {
+            "description": "Describes the error if the operation is not successful.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Subscription/supportplans/default": {
+      "put": {
+        "operationId": "PurchaseSupportPlan_Create",
+        "x-ms-examples": {
+          "PurchaseSupport": {
+            "$ref": "./examples/PurchaseSupportPlan.json"
+          }
+        },
+        "description": "The operation to purchase/convert/cancel support plan.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/supportPlanNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Normal response for a successful query.",
+            "schema": {
+              "$ref": "#/definitions/SupportPlan"
+            }
+          },
+          "202": {
+            "description": "Accepted. Purchase support plan is in progress.",
+            "headers": {
+              "Location": {
+                "description": "GET this URL to retrieve the status of the asynchronous operation.",
+                "type": "string"
+              },
+              "Retry-After": {
+                "description": "The amount of delay to use while the status of the operation is checked. The value is expressed in seconds.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
```

Comment:
> " [](start = 7, length = 2)


Indicate on how the async operation completion will be notified.. 

``` json
"x-ms-long-running-operation-options": {
  "final-state-via": "location"
}
```

or

``` json
"x-ms-long-running-operation-options": {
  "final-state-via": "azure-async-operation"
}
```

Reply:
done